### ISSUE TITLE
Full Name user field missing & Editing First + Last not saving

### ISSUE THREAD

**Issue by vonscriptor** [solspace/craft3-freeform#1066]
**Describe the bug or issue you're experiencing**
On existing or new forms in the Element Connections tab for a Users type I do not see a Field mapping for the "Full Name" Craft Field.

I do see First and Last Name fields, however those are no longer saving values to user accounts. (I'm not sure when this stopped working). Other fields are saving properly.

Front-end implementation:
```
{% if currentUser %}
    {{ craft.freeform.form("accountSettings").render({
        elementId: currentUser.id
    }) }}
{% endif %}
```

**Steps to reproduce**

1. Create or edit a form
2. Add a Users Type Element Connections
3. The Craft Field "Full Name" is missing

**Expected behavior**
Full Name Craft Field should appear for Element Connections Field mapping. First Name and Last Name fields should save values to the user's account.

- Craft Version: 4.7.1
- Freeform Version: 4.1.13
- Freeform Edition: Pro

**Additional context**
At one point I did have general.php `'showFirstAndLastNameFields' => true`, but enabled or not it doesn't seem to correct the issue. It might have something to do with how the issue started happening though.

**Comment by kjmartens** [comment]
Sorry for the trouble, @vonscriptor.

From what I can see, the primary issue is that attempting to **EDIT** the First and Last Name (or presumably the Full Name) is not working correctly when using the `elementId: currentUser.id` approach. I was, however, able to create a new user with the first and last name successfully.

I will queue this up for a developer to address the issue of it not updating the name upon edit. I will also see if we can work in an additional "Full Name" field mapping too for those with that preference. 🙂

**Comment by vonscriptor** [comment]
Thanks for investigating and confirming. The "Full Name" field seems to be Craft's preferred user field. Additional info: https://github.com/craftcms/cms/issues/12305#issuecomment-1315643772

The config setting `showFirstAndLastNameFields` for replacing “Full Name” fields with “First Name” and “Last Name” was only recently added in Craft CMS 4.6.0: https://github.com/craftcms/cms/releases/tag/4.6.0

**Comment by kjmartens** [comment]
Yes, I agree that seems to be the preferred approach, so it makes sense for us to support it. Thanks for tracking down which version made this change... I was curious. 😄

**Comment by kjmartens** [comment]
This is fixed now in Freeform 5.0.13+. It includes a fix to updating existing users, as well as the ability to map directly to the **Full Name** field. 🙂
